install/install.py

```diff
@@ -115,15 +116,16 @@ def install(self, cmd_arguments):
 
             print("")
             for i in range(0, len(cp.sections())):
-                print("(%d) %s"%(i+1, cp.sections()[i]))
-            print("(0) %s (%s)"%("My distro is not listed :(", distro))
-            distro_num = raw_input("Select a number based on your distribution : ")
+                Colorizer.warning("(%d) %s" % (i + 1, cp.sections()[i]))
```

Comment:
I didn't even see. Please update the ugly C-style for loop to something more pythonic with `enumerate`.